### PR TITLE
browser test

### PR DESCRIPTION
**Pull request by KarpitPatel6004**

### CODE REVIEW

**Comment by KarpitPatel6004** [comment]
I have deleted the files related to profile picture test case.
so, you can merge it.

File: Browser_test/user/test_register.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+web.init();
+web.open('https://setnow.herokuapp.com/');
```
Comment (Niraj-Kamdar) [inline Browser_test/user/test_register.js:2]: use localhost for testing

File: test/__init__.py [hunk 2]
```diff
@@ -1 +1,3 @@
+import os
 
+TEST_PATH = os.path.dirname(os.path.abspath(__file__))
```
Comment (Niraj-Kamdar) [inline test/__init__.py:3]: remove this file, irrelevant for this pr

**Comment by Niraj-Kamdar** [comment]
Make a function for registration process

**Comment by KarpitPatel6004** [comment]
"Make a function for registration process."
you mean unit test ?
i got javascript file from oxygen software.

**Comment by Niraj-Kamdar** [comment]
don't mind just do login

**Comment by Niraj-Kamdar** [comment]
use localhost strictly

**Comment by Niraj-Kamdar** [comment]
You aren't asserting anything assert something like flash message You have registered succesfully there is an extention for chrome browser that does that see video for more info

**Comment by Niraj-Kamdar** [comment]
assert something

**Pull request closed without merging**